src/components/Explore/actions/AddToFiltersAction.test.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react'; 
+import { AddToFiltersAction, addToFilters } from './AddToFiltersAction';
+import { DataFrame } from '@grafana/data';
+import { AdHocFiltersVariable } from '@grafana/scenes';
+
+jest.mock('../../../utils/utils', () => ({
```

Comment:
```suggestion
import { getFiltersVariable, getLabelValue } from '../../../utils/utils';

jest.mock('../../../utils/utils');
```

Reply:
Nice spot, they're actually not needed in the mock.